packages/attention/index.js

```diff
@@ -135,23 +135,25 @@ class WarpAttention extends kebabCaseAttributes(LitElement) {
 
   get _wrapperClasses() {
     return classes({
-      [c.base]: true,
-      [c.tooltip]: this.tooltip,
-      [c.callout]: this.callout,
-      [c.popover]: this.popover,
+      [ccAttention.base]: true,
+      [ccAttention.tooltip]: this.tooltip,
+      [ccAttention.callout]: this.callout,
+      [ccAttention.popover]: this.popover,
     });
   }
 
   get _arrowClasses() {
+    const arrowType = () => {
+      if (this.tooltip) return ccAttention.arrowTooltip;
+      else if (this.callout) return ccAttention.arrowCallout;
+      else if (this.popover) return ccAttention.arrowPopover;
+      return '';
+    }
+
     return classes({
-      [c.arrowBase]: true,
-      [`-top-8`]: this._arrowDirection === 'top',
-      [`-right-8`]: this._arrowDirection === 'right',
-      [`-bottom-8`]: this._arrowDirection === 'bottom',
-      [`-left-8`]: this._arrowDirection === 'left',
-      [c.arrowTooltip]: this.tooltip,
-      [c.arrowCallout]: this.callout,
-      [c.arrowPopover]: this.popover,
+      [ccAttention.arrowBase]: true,
+      [ccAttention[`arrowDirection${this._arrowDirection.charAt(0).toUpperCase() + this._arrowDirection.slice(1)}`]]: true,
+      [arrowType()]: this.tooltip || this.callout || this.popover,
```

Comment:
Would have preferred the way it was before here...? 🙈😅 A lot clearer to me. Was there anything problematic with that? 🙂